### DISCUSSION TITLE
SwiftUI and Swift and Objective C : getting multiple error while adding pod files into project

### DISCUSSION THREAD

**Discussion by SnehalVJ26** [ngageoint/projections-ios#11] [Q&A]
Hello @bosborn and @danielbarela  and  Everyone ,
I am trying to add the below framework into driver app for Japan coordinate calculation.
https://github.com/ngageoint/projections-ios?tab=readme-ov-file
I have followed all the steps. But when I install the pod and run the app then it gives error like` 'crs.hpp' file not found with <angled> include; use "quotes" instead`
So I have checked the discussion and try other steps like replacing use_frameworks! with use_modular_headers!..but it gives another error like
`module map file '/Users/snehaljadhav/SourceTree/Wise_Delivery/Pods/Headers/Private/grpc/gRPC-Core.modulemap' not found`
.
Actually previously we were using use_frameworks!  and it having dependency for other library. Now I have removed use_frameworks! and added use_modular_headers! do it gives error to other pod library.

I have tried multiple ways but not able to build the app successfully after adding above framework in pod file.
So just wanted to check with you guys if you have any idea or solution for this. It would be really appreciated.

 Thank you!

**Comment by danielbarela** [comment]
In one of our other projects, we do not have use_frameworks in our Podfile but we had to add a special modular_headers option to one of the other libraries we use: 
https://github.com/ngageoint/marlin-ios/blob/1886ea338d0f8b92bc45ef409e36ba663d0894e3/Marlin/Podfile#L26
